packages/@aws-cdk/aws-ecr-assets/lib/image-asset.ts

```diff
@@ -39,6 +39,13 @@ export interface DockerImageAssetProps extends assets.CopyOptions {
    * @default - no target
    */
   readonly target?: string;
+
+  /**
+   * Path to the Dockerfile (relative to the directory).
+   *
+   * @default - 'Dockerfile'
```

Comment:
since the default is an actual value, you should just put it (in quotes) instead of “-“. We use the hyphen in cases where the default is only described behaviorally.
```suggestion
   * @default 'Dockerfile'
```